pkg/controller/autoscaler/horizontalpodautoscaler_controller.go

```diff
@@ -40,41 +35,20 @@ const (
 )
 
 type HorizontalPodAutoscalerController struct {
```

Comment:
The full name of this type is `autoscalercontroller.HorizontalPodAutoscalerController`. That is really a mouthful. Again, don't need to change in this PR.

@jszczepkowski please take a look at my naming comments here. Also see my comment here: https://github.com/kubernetes/kubernetes/pull/13278/files#r38226042